sdk/java/pulumi/src/main/java/com/pulumi/experimental/automation/ProjectRuntimeOptions.java

```diff
@@ -11,14 +11,32 @@ public class ProjectRuntimeOptions {
     @Nullable
     private final Boolean typescript;
     @Nullable
+    private final String nodeargs;
+    @Nullable
+    private final String packagemanager;
```

Comment:
Guessing this casing is deliberate to get serde "for free"?

Reply:
Yes